### ISSUE TITLE
Interface suspended animation problem report

### ISSUE THREAD

**Issue by foxiexianmo** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2386]
It is not clear what causes the interface to enter a state of suspended animation.At this time, the interface is not really unresponsive, but it feels that the interface drawing mechanism has lost its function.I hope this problem can be solved, thank you.
This is the playback address of the video record at that time:https://youtu.be/j-FBJmvWfwI

**Comment by Xaalek** [comment]
I have the similar issue when I keep the app openned and come back like an hour later.

**Comment by foxiexianmo** [comment]
Yes, I thought it was the screen saver at first, but later found that it would appear on different versions of win10 systems, and finally found more than this, it seems that all versions of the system will appear, but the probability of appearing is very small, and maximized after opening the program Related to minimizing restoration, it seems that the background mechanism during redrawing has collapsed.

**Comment by Xaalek** [comment]
Maybe it's because we are using `<WindowChrome.WindowChrome>`

**Comment by foxiexianmo** [comment]
There are reports on the Internet that WindowChrome can cause this phenomenon? Do you have any solutions?

**Comment by Xaalek** [comment]
Unfortunately I don't. I hope someone could help us

**Comment by Keboo** [comment]
In the sample video, are you also using MahApps (if so which version) or just custom WindowChrome?

Internally MahApps uses ControlzEx for the window chrome interaction and there is lots of [WinProc stuff there](https://github.com/ControlzEx/ControlzEx/tree/develop/src/ControlzEx/Microsoft.Windows.Shell). I suspect the issue lies there and not in this library.

**Comment by Erapchu** [comment]
@Keboo, this is known issue #2260. To solve it you @foxiexianmo should DISABLE Shadow Cache. So, please ensure that you disable it in this window and you don't have the same windows (even if they're hidden).
Instead of using BitmapCache, we should move to other Shadow effect #2377.

In window style (can be setted as you wish):
`<Setter Property="materialDesign:ShadowAssist.CacheMode" Value="{x:Null}"/>`

**Comment by Erapchu** [comment]
The problem with BitmapCache, not with WindowChrome or other components. Author can leave the same window style, but extend it for current window(s) by set this property. Please, additionally ensure that you don't have controls with directly defined in XAML BitmapCache. If it exists, you need to re-write the xaml code to use it without BitmapCache.

To reproduce: https://github.com/Erapchu/GhostWindows (WPF windows without any additional NuGets or styles)
